### PR TITLE
thirdweb engine landing page updates

### PR DESCRIPTION
**Pull request by easyrun42**

### CODE REVIEW

**Comment by jnsdls** [comment]
#### 📝 Changed routes:


| [**/engine**](https://thirdweb-4p2tar45c.thirdweb-preview.com/engine) |
|:-:|
|<a href="https://thirdweb-4p2tar45c.thirdweb-preview.com/engine">Screenshot of /engine</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/v1Ax_CdgTNrnYGh5p6oGQkSz8f9YIOGrtnADWIoy2Gd3ZkV34KfOA1NUhAGptqpjy4wNSD8nDSezYFXztZEJ3A.png">(view full size)</a> |







Commit c2f7245bf6122ff5a5e3d85d1f7f006ea23fd631 (https://thirdweb-4p2tar45c.thirdweb-preview.com).

File: pages/engine.tsx [hunk 1]
```diff
@@ -40,30 +79,34 @@ const EngineLanding: ThirdwebNextPage = () => {
...
-          noContactUs={true}
-          gradient="linear(to-r, #BFA3DA, #84309C, #C735B0)"
+          ctaLink="https://portal.thirdweb.com/engine"
+          ctaText="Get started"
+          miniTitleSize="subtitle.lg"
+          contactUsTitle={"Book a demo"}
+          gradient="linear-gradient(244deg, #BFA3DA -84.5%, #FFF -84.49%, #3385FF 86.64%)"
           image={require("public/assets/product-pages/engine/desktop-hero.png")}
           mobileImage={require("public/assets/product-pages/engine/desktop-hero.png")}
+          contactUsLink={"/contact-us"}
```
Comment (nachoiacovino) [inline pages/engine.tsx:93]:
```suggestion
          contactUsLink={"/contact-us"}
```
this is default so no needed

File: pages/engine.tsx [hunk 2]
```diff
@@ -40,30 +79,34 @@ const EngineLanding: ThirdwebNextPage = () => {
...
+          title="All-in-one API for enterprise-grade Web3 apps"
+          subtitle="Onboard all of your users with a powerful Connect Wallet modal, flexible sign-in options for web2 & web3, and hooks for full customizability."
           trackingCategory={TRACKING_CATEGORY}
-          ctaLink="https://share.hsforms.com/1b5uu_0bSQ3GX5NCQyrIeGAea58c"
-          ctaText="Get beta access"
-          noContactUs={true}
-          gradient="linear(to-r, #BFA3DA, #84309C, #C735B0)"
+          ctaLink="https://portal.thirdweb.com/engine"
+          ctaText="Get started"
+          miniTitleSize="subtitle.lg"
```
Comment (nachoiacovino) [inline pages/engine.tsx:88]: is this something we want to do always? like I feel like we need to stay consistent with the miniTitle so if we want to switch to subtitle.lg that's fine but I don't think it should be only for this page

File: pages/engine.tsx [hunk 3]
```diff
@@ -40,30 +79,34 @@ const EngineLanding: ThirdwebNextPage = () => {
...
+          contactUsTitle={"Book a demo"}
```
Comment (nachoiacovino) [inline pages/engine.tsx:89]:
```suggestion
          contactUsTitle="Book a demo"
```
nitpick but we don't need braces here

File: pages/engine.tsx [hunk 4]
```diff
@@ -88,61 +131,148 @@ const EngineLanding: ThirdwebNextPage = () => {
...
+              bgColor="#FFF"
+              color="#000"
+              border="none"
+              _hover={{
+                background: "#e6d8d8",
+              }}
+              py={6}
+              category={TRACKING_CATEGORY}
+              label="starter"
+              href="/dashboard/settings/billing"
```
Comment (nachoiacovino) [inline pages/engine.tsx:263]: this links is wrong? :D

File: pages/engine.tsx [hunk 5]
```diff
@@ -88,61 +131,148 @@ const EngineLanding: ThirdwebNextPage = () => {
...
+            linkBoxBg={linkBoxBg}
+            colSpan={1}
+          />
+        </LandingGridSection>
+
+        <LandingCardWithImageBackground
+          image={require("public/assets/landingpage/coinbase-event.png")}
+        >
+          <Box maxWidth="600px">
+            <Heading fontSize="32px" fontWeight="600" color="white">
```
Comment (nachoiacovino) [inline pages/engine.tsx:237]: same thing let's not hardcode fontSizes

**Review by nachoiacovino — CHANGES_REQUESTED** [review]
there are some nitpicks but some other things we should be consistent with the rest of the existing code

File: components/landing-pages/card-with-image-background.tsx [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+interface CardWithImageProps {
+  image: string;
+  children: ReactNode;
+}
+
+const LandingCardWithImageBackground = ({
+  children,
+  image,
+}: CardWithImageProps) => {
```
Comment (easyrun42) [inline components/landing-pages/card-with-image-background.tsx:13]: This is the component that's used for the `coinbase` section

**Pull request merged**